src/import-export/azext_import_export/generated/action.py

```diff
@@ -0,0 +1,211 @@
+        d = {}
+        for k in properties:
+            kl = k.lower()
+            v = properties[k]
+            if kl == 'carrier-name':
+                d['carrier_name'] = v
+            elif kl == 'tracking-number':
+                d['tracking_number'] = v
+            elif kl == 'drive-count':
+                d['drive_count'] = v
+            elif kl == 'ship-date':
+                d['ship_date'] = v
+        return d
+
+
+class AddReturnPackage(argparse.Action):
+    def __call__(self, parser, namespace, values, option_string=None):
+        action = self.get_action(values, option_string)
+        namespace.return_package = action
+
+    def get_action(self, values, option_string):  # pylint: disable=no-self-use
+        try:
+            properties = dict(x.split('=', 1) for x in values)
+        except ValueError:
+            raise CLIError('usage error: {} [KEY=VALUE ...]'.format(option_string))
+        d = {}
+        for k in properties:
+            kl = k.lower()
+            v = properties[k]
+            if kl == 'carrier-name':
+                d['carrier_name'] = v
+            elif kl == 'tracking-number':
+                d['tracking_number'] = v
+            elif kl == 'drive-count':
+                d['drive_count'] = v
+            elif kl == 'ship-date':
+                d['ship_date'] = v
+        return d
+
+
+class AddDriveList(argparse._AppendAction):
+    def __call__(self, parser, namespace, values, option_string=None):
+        action = self.get_action(values, option_string)
+        super(AddDriveList, self).__call__(parser, namespace, action, option_string)
+
+    def get_action(self, values, option_string):  # pylint: disable=no-self-use
+        try:
+            properties = dict(x.split('=', 1) for x in values)
+        except ValueError:
+            raise CLIError('usage error: {} [KEY=VALUE ...]'.format(option_string))
+        d = {}
+        for k in properties:
+            kl = k.lower()
+            v = properties[k]
+            if kl == 'drive-id':
+                d['drive_id'] = v
+            elif kl == 'bit-locker-key':
+                d['bit_locker_key'] = v
+            elif kl == 'manifest-file':
+                d['manifest_file'] = v
+            elif kl == 'manifest-hash':
+                d['manifest_hash'] = v
+            elif kl == 'drive-header-hash':
+                d['drive_header_hash'] = v
+            elif kl == 'state':
+                d['state'] = v
+            elif kl == 'copy-status':
+                d['copy_status'] = v
+            elif kl == 'percent-complete':
+                d['percent_complete'] = v
+            elif kl == 'verbose-log-uri':
+                d['verbose_log_uri'] = v
+            elif kl == 'error-log-uri':
+                d['error_log_uri'] = v
+            elif kl == 'manifest-uri':
+                d['manifest_uri'] = v
+            elif kl == 'bytes-succeeded':
+                d['bytes_succeeded'] = v
```

Comment:
Can be simplified.
And where do these string literals come from? Are they defined in SDK? If so, avoid so many string literals.